test/stress_tests/run_stress_tests.sh

```diff
@@ -33,3 +33,4 @@ pushd "$ROOT_DIR"
 popd
 
 cat $RESULT_FILE
+[ ! -s $RESULT_FILE ] || exit 1
```

Comment:
Is this a bug fix?

Reply:
Yes otherwise test basically always passes (assuming it runs)